apps/demos/Demos/Chat/MessageEditing/Vue/App.vue

```diff
@@ -107,15 +108,17 @@ const onMessageUpdated = ({ message, text }) => {
   ]);
 };
 
-const editingStrategy = {
+const editingStrategy: Record<string, boolean | ((options: CustomStrategyOptions) => boolean)> = {
   enabled: true,
   disabled: false,
-  custom: ({ component, message }) => {
+  custom: ({ component, message }: CustomStrategyOptions) => {
+    if (!component) return;
+    
     const { items, user } = component.option();
-    const userId = user.id;
+    const userId = user?.id;
 
-    const lastNotDeletedMessage = items.findLast(
-      (item) => item.author?.id === userId && !item.isDeleted
+    const lastNotDeletedMessage = items?.findLast(
+      (item: any) => item.author?.id === userId && !item.isDeleted
```

Comment:
https://js.devexpress.com/Vue/Documentation/ApiReference/UI_Components/dxChat/Types/Message/
```suggestion
      (item:Message) => item.author?.id === userId && !item.isDeleted
```